backend/service/topology/cache.go

```diff
@@ -91,16 +91,20 @@ func (c *client) startTopologyCache(ctx context.Context) {
 					c.log.Error("Unable to start topology caching", zap.String("service", n), zap.Error(err))
 					continue
 				}
-				go c.processTopologyObjectChannel(ctx, topologyChannel)
+				go c.processTopologyObjectChannel(ctx, topologyChannel, n)
```

Comment:
nit: rename `n`. when the var is this far from the declaration a longer name helps readability